game-core/src/main/java/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -648,19 +672,35 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
         .build();
   }
 
-  public static Predicate<Unit> defendingBombingRaidInterceptors(final PlayerID attacker, final GameData data) {
-    return PredicateBuilder.of(
+  /**
+   * Returns a unit predicate that determines if it can potentially intercept including checking any
+   * air base requirements.
+   */
+  public static Predicate<Unit> defendingBombingRaidInterceptors(final Territory territory, final PlayerID attacker,
```

Comment:
I'm noting that this method is used within this class and from another. In both cases the behavior is dependent on a 'territory', would it make sense to move this to 'Territory'? I think the benefits there would be:
- the static method would become stateful
- one less parameter
- potentially easier to find this functionality potentially attached to territory, rather than a utility class for AirBattle
- fewer parameters generally make for easier testing.
- would avoid a public method from being called from within the same class.

Reply:
I'm not 100% sure Territory would be the right home. The other parameters are candidates for a stateful home. I am pretty okay with the static utility method being here if it is not to be attached to an object where it does not really belong, but the engineering part of me really wonders if this can be made stateful (and presumably also cleaner and easier to test)

Reply:
Yeah, I don't think `Territory` would be the proper place. If anything maybe push it to `Matches`. But this is where it was as I just updated the method and I think its ok here.